kbase-extension/static/kbase/js/common/runtime.js

```diff
@@ -56,6 +56,7 @@ define([
         // These are still module scope
 
         function authToken() {
+            //Failing line during tests.
             return Jupyter.narrative.getAuthToken();
```

Comment:
When the FE unit tests fail, `runtime.authToken` is called and error occurs because `Jupyter.narrative` is not defined.

What I don't know:
* Why does this execute when there are faster processing/internet speeds?
* How does `Jupyter.narrative` get created during the FE unit tests?

This is the root issue I have found so far.